.utility/update-docs-repo.sh

```diff
@@ -13,7 +13,7 @@ git clone --quiet --branch=master git@github.com:loklak/dev.loklak.org.git lokla
 
 cd loklak_docs
 
-if ! git subtree pull --prefix=raw/server ../loklak_server documentation --squash -m "Update server subtree"
+if ! git subtree pull --prefix=raw/server ../loklak_server documentation --squash -m "Update server subtree" ; then
   git subtree add --prefix=raw/server ../loklak_server documentation --squash -m "Update server subtree"
 fi
```

Comment:
Won't this end up giving an error again? Because when you check for proper execution of `git subtree...`, it will already have created the directory and you'll be recreating it in the body of the if statement?

In other words, if `if` passes, i.e. `git subtree ...` fails, you try to run `git subtree ...` again, which would fail again (?). If all you want to do is to ignore the case if any error occurs, you may do something like this - 
```bash
git subtree ... || true
```

I may be getting something wrong here, please let me know in that case.